src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/helper.py

```diff
@@ -0,0 +1,361 @@
+import json
+import logging
+import time
+from pathlib import Path
+from typing import Any, Optional
+
+import requests
+from requests import Response
+from requests.exceptions import ConnectionError, HTTPError, Timeout
+
+from unstract.sdk.adapters.exceptions import ExtractorError
+from unstract.sdk.adapters.utils import AdapterUtils
+from unstract.sdk.adapters.x2text.llm_whisperer_v2.src.constants import (
+    HTTPMethod,
+    Modes,
+    OutputModes,
+    WhispererConfig,
+    WhispererDefaults,
+    WhispererEndpoint,
+    WhispererHeader,
+    WhisperStatus,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class LLMWhispererHelper:
```

Comment:
Not reusing the code from v1 because,
1. To maintain the mutual exclusivity of adapters.
2. V1 will be eventually deprecated.